server/db/schema.sql

```diff
@@ -12,23 +12,22 @@ CREATE TABLE reviews (
   body TEXT,
   name VARCHAR(50),
   email VARCHAR(100)
```

Comment:
In case you are unsure about the number of characters for your email, you can always use the type TEXT instead of VARCHAR(100), but VARCHAR works here too!

Reply:
gotcha! I thought I would waste more space from that so yeah idk